element/index.js

```diff
@@ -35,6 +43,15 @@ import serialize from './serialize';
  */
 export { createElement };
 
+/**
+ * Returns an object tracking a reference to a rendered element via its
+ * `current` property as either a DOMElement or Element, dependent upon the
+ * type of element rendered with the ref attribute.
+ *
+ * @return {Object} Ref object.
+ */
+export { createRef };
```

Comment:
We still need to add `forwardRef` - it can be done in another PR.

Reply:
>We still need to add `forwardRef` - it can be done in another PR.

Yeah, I suppose we'd want to build this into `createHigherOrderComponent` ?

Reply:
RichText with Autocomplete has a few wrappers with refs, so we can check there. I guess HOC makes the most sense in our case.